FieldOpt/ERTWrapper/eclgridreader.h

```diff
@@ -61,24 +61,16 @@ class ECLGridReader
   */
   struct Cell {
     int global_index;
-    float volume;
+    bool active;
+    double volume;
+    double porosity;
+    double permx;
+    double permy;
+    double permz;
     std::vector<Eigen::Vector3d> corners;
     Eigen::Vector3d center;
   };
 
-  /*!
-   * @brief The CellProperties struct contains all non-geometric properties
```

Comment:
This text can be transferred to the cell \brief above:
-   * The cell struct also contains all non-geometric properties
 -   * for a cell, i.e. permeabilities, porosities, and whether or 
 -   * not the cell is active.